public/components/data_connections/components/__tests__/testing_constants.ts

```diff
@@ -55,3 +55,727 @@ export const describeDataConnection = {
     'emr.cluster': 'j-3UNQLT1MPBGLG',
   },
 };
+
+export const mockRoleData = {
```

Comment:
you can move the test constants here: https://github.com/opensearch-project/dashboards-observability/tree/main/test

Reply:
Done in: [9ef64f2](https://github.com/opensearch-project/dashboards-observability/pull/1024/commits/9ef64f2dec4c065ad86757013dcaf092cb3cd72f)